.rultor.yml

```diff
@@ -0,0 +1,2 @@
+merge:
+  script: echo "Nothing to do (yet)."
```

Comment:
It worth adding a configuration that does nothing? Maybe you can add a simple (but useful) script here?
This is not a blocking, just a suggestion.